### PR TITLE
security manager for so auth

### PR DESCRIPTION
**Pull request by ankitkailaswar**

### CODE REVIEW

**Comment by dpgaspar** [comment]
Hi @ankitkailaswar,

Thanks, but could you add some details and updated the docs for this?

File: flask_appbuilder/security/views.py [hunk 1]
```diff
@@ -556,3 +561,78 @@ def login(self):
...
+import ssl
+import jwt
+
+class AuthSSOView(AuthView):
+    login_template = 'appbuilder/general/security/login_ldap.html'
+    logout_template = 'appbuilder/general/security/logout_sso.html'
+
+    @expose('/logout/')
+    def logout(self):
+        log.debug("log out user from caravel")
```
Comment (mrhlee) [inline flask_appbuilder/security/views.py]: Do we want to mention caravel (or superset)? This is Flask App Builder

**Comment by mistercrunch** [comment]
The build in master was broken for a while but has now been fixed. Please rebase or merge to pass the build.

**Comment by dpgaspar** [comment]
Hi @ankitkailaswar,

Thanks, could you please provide a description, documentation, and remove references from caravel or superset.

**Pull request closed without merging**